templates/bundle/resources/usr/local/bin/liferay_entrypoint.sh

```diff
@@ -83,4 +85,19 @@ function start_monitor_liferay_lifecycle {
 	fi
 }
 
+function start_interval_thread_dump {
+	while [ ! -z "${LIFERAY_DOCKER_AUTO_THREAD_DUMP_SLEEP_FILE}" ]
+	do
+		generate_thread_dumps_now
+	done &
```

Comment:
The & here is not really visible, please put it where we call this method, it way be cleaner that way.